google-cloud-bigquerydatatransfer/src/main/java/com/google/cloud/bigquery/datatransfer/v1/stub/DataTransferServiceStubSettings.java

```diff
@@ -712,7 +712,7 @@ private static Builder initDefaults(Builder builder) {
 
       builder
           .deleteTransferConfigSettings()
-          .setRetryableCodes(RETRYABLE_CODE_DEFINITIONS.get("idempotent"))
+          .setRetryableCodes(RETRYABLE_CODE_DEFINITIONS.get("non_idempotent"))
```

Comment:
I checked with the API team, this change is intended behavior. `DeleteTransfer`, `StartManualTransfer` and `DeleteTransferRun` are all non-idempotent methods.

Reply:
okay - got it